mmv1/templates/terraform/examples/vmware_engine_subnet_user_defined.tf.erb

```diff
@@ -0,0 +1,30 @@
+resource "google_vmwareengine_network" "subnet-nw" {
+  name        = "<%= ctx[:vars]['network_id'] %>"
+  location    = "global"
+  type        = "STANDARD"
+  description = "PC network description."
+}
+
+resource "google_vmwareengine_private_cloud" "subnet-pc" {
+  location    = "<%= ctx[:test_env_vars]['region'] %>-a"
```

Comment:
`region` needs to be aded to `test_env_vars` in yaml file.

Reply:
Done.